content/courses/advanced-ada/chapters/statements.rst

```diff
@@ -3,6 +3,133 @@ Statements
 
 .. include:: ../../global.txt
 
+Simple and Compound Statements
+------------------------------
+
+We can classify statements in simple and compound statements. Simple statements
+don't contain other statements; think of them as "atomic units" that cannot be
+further divided. Compound statements, on the other hand, may contain other
+|mdash| simple or compound |mdash| statements.
+
+Here are some examples from each category:
+
++---------------------+--------------------------------------------------------+
+| Category            | Examples                                               |
++=====================+========================================================+
+| Simple statements   | Null statement, assignment, subprogram call, etc.      |
++---------------------+--------------------------------------------------------+
+| Compound statements | If statement, case statement, loop, block statement    |
++---------------------+--------------------------------------------------------+
+
+.. admonition:: In the Ada Reference Manual
+
+    - `5.1 Simple and Compound Statements - Sequences of Statements <http://www.ada-auth.org/standards/12rm/html/RM-5-1.html>`_
+
+Labels
+~~~~~~
+
+We can use labels to identify statements in the code. They are useful as a way
+to document the code or to indicate important statements in the code.
```

Comment:
I disagree with this claim. I've never seen or heard of them being used for those two purposes. They are only in the language, as far as I know, for being the targets of goto statements. If I came across one that wasn't a goto target I'd think it was very odd indeed.
One can make the case that goto statements have benefit as a way to construct the effect of "continue" statements in other languages, which Ada is missing.

Reply:
Thanks for the insight. I changed the text to focus more on `goto` statements.